tests/test_install.ts

```diff
@@ -125,7 +128,8 @@ describe("Agama test", function () {
     });
 
     it("should optionally display the product selection dialog", async function () {
-        this.timeout(60000);
+        let timeout = 5 * 60 * 1000;
```

Comment:
if this is 60 seconds, don't need to have the reader multiplying factor mentally, leave it as 60000.

Reply:
it's 5 minutes and it is not good practice to use magic numbers. I suggest to use a variable with each number to know exactly what each do.

Reply:
I didn't need 5 minutes, with 1min runs fine, are you sure we need that extra time?

I think it should be some fancy way to express that that is more readable, with types in TS, we can focus on that later, fine if you want to leave it like that, only check if really needed as my comment above.

Reply:
I tested it with slow connectivity and sometimes it takes more than 5 minutes, the matter is to take the mean value that will work most of the cases. If in this test you are looking for performance it's ok to low this value. If it is a general test like I think it is I vote to cover the majority of cases.

Reply:
the problem with the timeout is that other developers tends to add them all over the place and keep increasing it so I'm always pushing for going to the minimum and then see what specific parts we need more timeout, otherwise we leave plenty of places where it could be a potential bug because it is stucks too much time even if at the end reach the right screen. I would suggest for now if you could do your own adjustments in case you have slow connectivity would be better. Anyway we will update this depending how it behaves in openQA.

Reply:
Solved 1a30482